scripts/handle-user-data

```diff
@@ -35,11 +35,12 @@ mount_data_partition() {
     mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
     ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
 
-    # Here we used bind mount because currently openshift doesn't work
```

Comment:
mount_data_partition is only called then the disk gets prepared the first time right?